src/carwash/CarWashService.java

```diff
@@ -0,0 +1,77 @@
+package carwash;
+
+import car.Car;
+import car.CarOwner;
+import exceptions.BlackListedException;
+import exceptions.NegativeBalanceException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CarWashService {
+    private int  totalWashes= 0;
+    private double moneyEarned= 0.0 ;
+    private final List<CarOwner> blackList= new ArrayList<>();
+    private final WashStrategy washStrategy;
+
+    public CarWashService(WashStrategy washStrategy){
+        this.washStrategy = washStrategy;
+    }
```

Comment:
Add a default constructor without arguments and a setter for `washStrategy`.  Then make this field non-final